dom.go

```diff
@@ -133,10 +134,25 @@ func (h *HTML) restoreHTML(prev *HTML) {
 
 	// Styles
 	style := h.Node.Get("style")
-	for name, value := range h.styles {
-		oldValue := prev.styles[name]
-		if value != oldValue {
-			style.Call("setProperty", name, value)
+	for propName, prop := range h.styles {
+		setProperty := func() {
```

Comment:
Generally speaking avoid helpers like these in this function at all costs. They make things arbitrarily hard to reason about.